word_guess.rb

```diff
@@ -0,0 +1,193 @@
+class WordGuess
+  def initialize
+    @word_array = ["DOG", "CAT", "PET", "HUT"]
+    @word = @word_array.sample              #chooses a random word
+    @letter_array = @word.split("")         #splits the random word's letter into an array
+    @dash_array = [ " _ ", " _ ", " _ "]
```

Comment:
How do you think you could use the length of the `@letter_array` to dynamically create the `@dash_array`?